flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/RemoteStreamEnvironment.java

```diff
@@ -112,34 +107,6 @@ public RemoteStreamEnvironment(String host, int port, String... jarFiles) {
 	 *            provided in the JAR files.
 	 */
 	public RemoteStreamEnvironment(String host, int port, Configuration clientConfiguration, String... jarFiles) {
-		this(host, port, clientConfiguration, jarFiles, null);
-	}
-
-	/**
-	 * Creates a new RemoteStreamEnvironment that points to the master
-	 * (JobManager) described by the given host name and port.
-	 *
-	 * @param host
-	 *            The host name or address of the master (JobManager), where the
-	 *            program should be executed.
-	 * @param port
-	 *            The port of the master (JobManager), where the program should
-	 *            be executed.
-	 * @param clientConfiguration
-	 *            The configuration used to parametrize the client that connects to the
-	 *            remote cluster.
-	 * @param jarFiles
-	 *            The JAR files with code that needs to be shipped to the
-	 *            cluster. If the program uses user-defined functions,
-	 *            user-defined input formats, or any libraries, those must be
-	 *            provided in the JAR files.
-	 * @param globalClasspaths
-	 *            The paths of directories and JAR files that are added to each user code
-	 *            classloader on all nodes in the cluster. Note that the paths must specify a
-	 *            protocol (e.g. file://) and be accessible on all nodes (e.g. by means of a NFS share).
-	 *            The protocol must be supported by the {@link java.net.URLClassLoader}.
-	 */
-	public RemoteStreamEnvironment(String host, int port, Configuration clientConfiguration, String[] jarFiles, URL[] globalClasspaths) {
```

Comment:
Same here, I don't think we can just remove this.

Reply:
Reverted.